app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
**OPTIONAL** - Please add the react-redux library to your project dependency. Please fix this 😄